src/javascript/binary/common_functions/form_validation.js

```diff
@@ -73,6 +74,7 @@ const Validation = (() => {
                             $parent.append($('<div/>', { class: `${error_class} ${hidden_class}` }));
                         }
                         field.$error = $parent.find(`.${error_class}`);
+                        field.$error_address = $parent.find(`.${error_address_class}`);
```

Comment:
why are we setting this for everyone? can we just set it for selectors address_line_1 and address_line_2?